src/OpenTK.GraphicsLibraryFramework/Interfaces/IGLFW.cs

```diff
@@ -1111,214 +1434,202 @@ internal interface IGLFW
         /// This function returns the current value of the raw timer, measured in 1 / frequency seconds.
         /// To get the frequency, call <see cref="GetTimerFrequency"/>.
+        /// </para>
         /// </summary>
         /// <returns>The value of the timer, or zero if an error occurred.</returns>
         /// <remarks>
+        /// <para>
         /// This function may be called from any thread.
-        ///
-        /// TODO: Possible errors include <see cref="ErrorCode.NotInitialized"/>.
+        /// </para>
+        /// <para>
+        /// Possible errors include <see cref="ErrorCode.NotInitialized"/>.
+        /// </para>
         /// </remarks>
-        /// <seealso cref="GetTimerFrequency"/>
         long GetTimerValue();
 
         /// <summary>
+        /// <para>
         /// This function returns the frequency, in Hz, of the raw timer.
+        /// </para>
         /// </summary>
         /// <returns>he frequency of the timer, in Hz, or zero if an error occurred.</returns>
         /// <remarks>
+        /// <para>
         /// This function may be called from any thread.
-        ///
-        /// TODO: Possible errors include <see cref="ErrorCode.NotInitialized"/>.
+        /// </para>
+        /// <para>
+        /// Possible errors include <see cref="ErrorCode.NotInitialized"/>.
+        /// </para>
         /// </remarks>
-        /// <seealso cref="GetTimerValue"/>
         long GetTimerFrequency();
 
         /// <summary>
+        /// <para>
         /// This function returns the window whose OpenGL or OpenGL ES context is current on the calling thread.
+        /// </para>
         /// </summary>
         /// <returns>The window whose context is current, or <c>null</c> if no window's context is current.</returns>
         /// <remarks>
+        /// <para>
         /// This function may be called from any thread.
-        ///
-        /// TODO: Possible errors include <see cref="ErrorCode.NotInitialized"/>.
+        /// </para>
+        /// <para>
+        /// Possible errors include <see cref="ErrorCode.NotInitialized"/>.
+        /// </para>
         /// </remarks>
-        /// <seealso cref="MakeContextCurrent"/>
         unsafe Window* GetCurrentContext();
 
         /// <summary>
+        /// <para>
         /// This function swaps the front and back buffers of the specified window
         /// when rendering with OpenGL or OpenGL ES.
+        /// </para>
+        /// <para>
         /// If the swap interval is greater than zero,
         /// the GPU driver waits the specified number of screen updates before swapping the buffers.
-        ///
+        /// </para>
+        /// <para>
         /// The specified window must have an OpenGL or OpenGL ES context.
         /// Specifying a window without a context will generate a <see cref="ErrorCode.NoWindowContext"/> error.
-        ///
-        /// This function does not apply to Vulkan. If you are rendering with Vulkan, see vkQueuePresentKHR instead.
```

Comment:
Why removed?

Reply:
I removed the Vulkan documentation because our bindings don't support Vulkan.